sw/device/lib/flash_ctrl.h

```diff
@@ -31,6 +39,8 @@ typedef struct mp_region {
   uint32_t base;
   /** Region config size. */
   uint32_t size;
+  /** Region partition size. */
+  uint32_t part;
```

Comment:
Everywhere you use `part` as a partition identifier, it should be as type `part_type_t` not `uint32_t`.

This also applies to the arguments to:
- `flash_page_erase`
- `flash_write`
- `flash_read`

Reply:
ah yes thanks for catching that.

Reply:
updated, please take a look.